python/cac_tripplanner/templates/partials/event-detail.html

```diff
@@ -21,28 +21,68 @@
         <div class="info-article-header-info">
             <div class="info-article-header-main">
                 <h2 class="info-article-title">{{ event.name }}</h2>
-                {% if event.destination %}
+                {% if event.first_destination %}
                 <h3 class="info-event-destination">
                     at
-                    <a class="info-event-destination-link"
-                        href="{% url 'place-detail' pk=event.destination.pk %}">
-                        {{ event.destination.name }}
+                    <!-- limit to the first four destinations, with JS to show the rest, if any -->
+                    {% with max_destinations=4 %}
+                    {% for event_dest in event.event_destinations.all %}
+                    <div class="{% if forloop.counter > max_destinations %}extra-destination hidden-extra{% endif %}">
+                        <a class="info-event-destination-link"
+                            href="{% url 'place-detail' pk=event_dest.destination.pk %}">
+                            {{ event_dest.destination.name }}
+                        </a>
+                        {% if event_dest.start_date and event_dest.end_date %}
+                        {% if not event.single_day %}
+                        <!-- only show dates for multi-day events -->
+                            <div class="info-event-date">
+                            {{ event_dest.start_date|date:"D M j" }}&nbsp;
```

Comment:
I don't understand what you mean by "the time range is backwards." If the end date/time precedes the start date/time, the admin interface will prevent saving the event destination with an error. What are the full start and end dates and times for this event destination? (If the end date is a different year, it might present this way in the interface.)

![image](https://user-images.githubusercontent.com/960264/65155772-d67c4d80-d9fb-11e9-832f-8fe093b8ca9e.png)

Reply:
That event did span the new year, but for a realistic example, take an event running from the evening of one day to the morning a few days later (like a music festival might).  It looks like this in the admin:
![image](https://user-images.githubusercontent.com/6598836/65254387-33ded000-daca-11e9-9c4b-0b8878321275.png)

And comes out like this in the front-end:
![image](https://user-images.githubusercontent.com/6598836/65254401-3d683800-daca-11e9-9497-cec5c83fe538.png)

Showing only the start date for multi-day events makes it look like it's a one-day event that runs backwards in time.  I think it should either show the range for both or not show the time range.  Though the former, or some variation, seems better, since otherwise there's no indication that it's multi-day.

Reply:
Oh, I see. I've added the missing date range for event destinations in the template. Thanks for the explanation.
![image](https://user-images.githubusercontent.com/960264/65267100-5977d380-dae2-11e9-97bf-5bb1a86e144d.png)